pkg/cache/sql/informer/listoption_indexer.go

```diff
@@ -41,6 +41,12 @@ var (
 	InvalidColumnErr = errors.New("supplied column is invalid")
 )
 
+const (
+	objectDBAlias = "o"
+	fieldDBAlias  = "f"
+	labelsDBAlias = "lt" // because "l" can look too much like the numeral "1"
```

Comment:
I'd suggest baking those aliases directly in SQL code, interpolation makes it a little bit more difficult to understand what's going on in my eyes

Reply:
ok.  I did that because I'm never thrilled to see an "o", "O", or "l" in code by itself, but I'll revert that.